app/containers/MassEnergizeSuperAdmin/Summary/NormalAdminHome.js

```diff
@@ -203,14 +203,14 @@ class NormalAdminHome extends PureComponent {
         {graph_data && <ActionsChartWidget data={graph_data || {}} />}
 
         <br />
-        <Grid md={12} style={{ display: "flex" }}>
+        <Grid md={12} className={classes.acti}>
```

Comment:
@abdullai-t  this uses grid, so if you put the container property here, it fixes everything 🤣 🤣  easily instead of +34 💀  🤣 🤣

Reply:
@frimpongopoku - I don't know what this means.  Should it be changed for simplicity, or is this adequate?

Reply:
Yes, for simplicity. Even though this works